libarchive/ffi.py

```diff
@@ -125,7 +125,7 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_int], c_int)
+ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_longlong)
 ffi('entry_set_filetype', [c_archive_entry_p, c_int], c_int)
```

Comment:
@pombredanne : I can write dummy 3.5 GB file to disk using simple for loop but I believe it will have bad impact on test performance.